tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -691,6 +722,7 @@ def is_locked(self):
         return self.marionette.execute_script('return window.wrappedJSObject.Service.locked')
 
     def lock(self):
+        GaiaData(self.marionette).set_setting('lockscreen.enabled', True)
```

Comment:
I'm not wholly thriled this leaves behind the side effect of toggling the setting. I'm not sure what we can or can't do about that, though, unless we have the option of disabling it while the lockscreen is still up without disruption.